src/components/dashboard/jumbotron/DboardJumbotron.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import CurrentJumbotronContainer from './CurrentJumbotronContainer';
+import UpcomingJumbotronContainer from './UpcomingJumbotronContainer';
+import NullJumbotron from './NullJumbotron';
+
+/**
+ *  DBoardJumbotron handles the logic for which dashboard jumbotron we are
+ *  displaying (an ongoing trip, countdown to next trip, or no
+ *  upcoming trips)
+ */
+
+export default class DboardJumbotron extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      displayCurrent: this.props.currTripInfo.current,
```

Comment:
See comment earlier about the use of state vs helper functions. I think this could be applied here too.